packages/admin/package.json

```diff
@@ -10,6 +10,10 @@
         "@material-ui/lab": "^4.0.0-alpha.57",
         "debounce": "^1.2.0",
         "lodash.isequal": "^4.5.0",
+        "lodash.get": "4.4.2",
+        "lodash.transform": "4.6.0",
+        "lodash.isobject": "3.0.2",
+        "lodash.isempty": "^4.4.0",
```

Comment:
please review these dependencies and remove unused ones

Reply:
i will remove the not needed pkgs :)